peer-ids/peer-ids.md

```diff
@@ -3,7 +3,7 @@
 ## Keys
 
 
-Everything is based on a very simple protobuf in [libp2p/go-libp2p-crypto/pb/crypto.proto#L5](https://github.com/libp2p/go-libp2p-crypto/blob/master/pb/crypto.proto#L5):
+Our key pairs are transmitted on the wire using a simple protobuf defined in [libp2p/go-libp2p-crypto/pb/crypto.proto#L5](https://github.com/libp2p/go-libp2p-crypto/blob/master/pb/crypto.proto#L5):
```

Comment:
the private key is _not_ transmitted on the wire; this is for saving the keys to disk.